iwyu.cc

```diff
@@ -2585,11 +2603,17 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
 
   const UsingDecl* GetUsingDeclarationOf(const NamedDecl* decl,
                                          const DeclContext* using_context) {
```

Comment:
I liked when you renamed this to `use_context`, I thought that was much better, can you make that change here too?